lib/rubocop/cop/style/sample.rb

```diff
@@ -91,11 +91,12 @@ def sample_size_for_two_args(first, second)
           second.int_type? ? second.to_a.first : :unknown
         end
 
-        def range_size(range_node) # rubocop:todo Metrics/CyclomaticComplexity
```

Comment:
@marcandre I see that you left this todo, but I have removed it because the extra conditions below blew got flagged by `Metrics/CyclomaticComplexity` and `Metrics/PerceivedComplexity`. Did this todo had a plan it with, is it fine to straight out ✂️ it?

Reply:
It wasn't a todo, more a "hey, this code seems fine as is and can not gain much readability by splitting it up". So don't hesitate to ✂️

Reply:
Thanks!